pennylane_lightning/src/simulators/lightning_kokkos/tests/Test_StateVectorLKokkos.cpp

```diff
@@ -135,7 +135,8 @@ TEMPLATE_PRODUCT_TEST_CASE("StateVectorKokkos::applyMatrix with a pointer",
     SECTION("Test with different number of wires") {
         using KokkosVector = typename StateVectorT::KokkosVector;
         const size_t num_qubits = 5;
-        for (size_t num_wires = 1; num_wires < num_qubits; num_wires++) {
+        const size_t max_wires = 2; // TODO: bring up to num_qubits - 1
```

Comment:
Anything to do here?

Reply:
That's the OPENMP bug (hitting `multiQubitOp`).